components/raftstore/src/store/peer.rs

```diff
@@ -3158,6 +3158,27 @@ where
             RAFT_APPLY_AHEAD_PERSIST_HISTOGRAM.observe(apply_ahead_delta as f64);
         }
         fail_point!("after_send_to_apply_1003", self.peer_id() == 1003, |_| {});
+        fail_point!(
+            "send_fake_apply_change_after_send_to_apply_1",
```

Comment:
Consider sending the message in tests to keep the code clean.

Try this in tests.

```rust
    cluster.get_apply_router(store_id);
```